### PR TITLE
Added enter handler for textbox #372

### LINKED ISSUE #372: Add enter handler for textbox
Add enter handler with similar logic `onChange` has.

Steps to follow:
1. Setup project for dev.
2. Find `textbox.tsx`.
3. Implement the feature.
4. Find `textbox.py` and showcase your new feature there.
5. Add needed unit tests in `textbox.test.tsx`.

### PR DESCRIPTION
**Pull request by haneenabdo**
Closes #372 
**The PR fulfills these requirements:** (check all the apply)

- [x] It's submitted to the `main` branch.
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `feat: Add a button #xxx`, where "xxx" is the issue number).
- [x] When resolving a specific issue, the PR description includes `Closes #xxx`, where "xxx" is the issue number.
- [ ] If changes were made to `ui` folder, unit tests (`make test`) still pass.
- [ ] New/updated tests are included

### CODE REVIEW

File: ide/tsconfig.json [hunk 1]
```diff
@@ -12,8 +12,12 @@
 		],
 		"types": [],
 		"baseUrl": "./node_modules",
+		"paths": {
+			"@fluentui/react": ["node_modules/@fluentui/react"]
+		  },
 		"jsx": "preserve",
 		"esModuleInterop": true,
+		"include": ["src", "node_modules/@types"],
```
Comment (mturoci) [inline ide/tsconfig.json:20]: Why is this needed?

File: ui/src/textbox.tsx [hunk 2]
```diff
@@ -67,6 +67,8 @@ export interface Textbox {
   spellcheck?: B
   /** Keyboard to be shown on mobile devices. Defaults to 'text'. */
   type?: 'text' | 'number' | 'tel'
+  /**T rue if the Enter button is being pressed */
+  onEnter?: B
```
Comment (mturoci) [inline ui/src/textbox.tsx:71]: We should probably do this by default, no need for an extra prop.

**Review by mturoci — CHANGES_REQUESTED** [review]
Thanks @haneenabdo! Can you pleases also include unit tests and a video demo of how your changes work in practice?

**Comment by michaeltraxler** [comment]
I would be really happy if this PR would be committed, as I also really need the feature, that pressing Enter in a textbox will call the handler (do something with the contents). An additional button just for this is really not what the people will expect and slow in our environment.

**Comment by mturoci** [comment]
Closing as inactive.

**Comment by Sachin-Ram** [comment]
is this issue still active?

**Pull request closed without merging**